modules/gapi/test/gapi_fluid_test_kernels.cpp

```diff
@@ -448,6 +449,43 @@ GAPI_FLUID_KERNEL(FSum2MatsAndScalar, TSum2MatsAndScalar, false)
     }
 };
 
+GAPI_FLUID_KERNEL(FSumArrayToMat, TSumArrayToMat, false)
+{
+    static const int Window = 1;
+    static const int LPI    = 2;
+
+    static void run(const std::vector<cv::Mat>    &arr,
+                          cv::gapi::fluid::Buffer &out)
```

Comment:
Does this work with fluid?

Reply:
I believe it shouldn't :)

Reply:
I'd keep this test here but mark it as DISABLED since we didn't design Fluid to handle cases like this.

Reply:
Removed that